app/services/profiles/get_prisoner_category_attributes.rb

```diff
@@ -0,0 +1,18 @@
+module Profiles
+  class GetPrisonerCategoryAttributes
+    def initialize(nomis_booking_id)
+      @nomis_booking_id = nomis_booking_id
+    end
+
+    def call
+      if @nomis_booking_id.present?
+        NomisClient::PrisonerCategory.get(@nomis_booking_id)
+      else
+        {
+          category: nil,
```

Comment:
I'm wondering whether this service is necessary? If we only merge in the attributes coming back from the client when there is a booking id then I think we get the same behaviour as if these attributes were nil? Same situation for the rake task I guess. And in fact, Looking at the ImportPrisonerCategory service that's exactly what you are doing :).


```ruby
attributes.merge(prisoner_category_attributes) if nomis_booking_id.present?
```

Reply:
yep, the `GetPrisonerCategoryAttributes` service isn't doing very much. I was trying to follow the existing pattern of calling the Nomis client through an intermediate service rather than directly in the controller. But I agree - it would be simply to just disband with this particular service - I'll remove it.